Transformations/utilities/testTimestamp.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+python AddTimeStamps.py \
+    ../test-data/RDF/historicalSuccession_CityGML_3.0_v1.ttl \
+    ./historicalSuccession_v1.ttl \
+    2000-01-01T00:00:00 2000-01-01T00:00:00 \
+    -d
+
+python AddTimeStamps.py \
+    ../test-data/RDF/historicalSuccession_CityGML_3.0_v2.ttl \
+    ./historicalSuccession_v2.ttl \
+    2001-01-01T00:00:00 2001-01-01T00:00:00 \
+    -d
+
+
+# python AddTimeStamps.py \
```

Comment:
Are the following commented lines important? If not, these could be improved.

Reply:
moved script to [readme](https://github.com/VCityTeam/UD-Graph/blob/rules/Transformations/utilities/README.md#addtimestamps)